### PR TITLE
UI cart

### PR DESCRIPTION
**Pull request by dungpt-0371**
## Related Tickets:
- [#68054](https://edu-redmine.sun-asterisk.vn/issues/68054)

## WHAT this PR does?
Tạo giao diện cho trang giỏ hàng

## Notes Impacted Areas:
- app/Models/CartController.php
- Routes/web.php
- views/cart

## Attachments:
image

### CODE REVIEW

**Comment by anhcnn-0378** [comment]
Oke

**Comment by dungpt-0371** [comment]
ready

File: app/Http/Controllers/CartController.php [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $user = Auth::user();
+        $cartItems = $user->carts;
```
Comment (ghost) [inline app/Http/Controllers/CartController.php]:
```suggestion
        $cartItems = Auth::user()->carts;
```

File: resources/views/cart/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            var totalElements = document.querySelectorAll('[id^="total-"]');
+            var cartTotal = 0;
+            totalElements.forEach(function(element) {
+                var text = element.textContent.replace('$', '');
+                cartTotal += parseFloat(text);
+            });
+            document.getElementById('cart-total').textContent = 'Tổng tiền: $' + cartTotal.toFixed(2);
+        }
+    </script>
+</x-app-layout>
```
Comment (ghost) [inline resources/views/cart/index.blade.php]: eof

**Comment by dungpt-0371** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**